libs/stokesian_dynamics/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+if(STOKESIAN_DYNAMICS_GPU)
+  add_gpu_library(sd_gpu SHARED sd_gpu.cu)
+  target_link_libraries(sd_gpu PRIVATE
+    ${CUDA_CUBLAS_LIBRARIES}
+    ${CUDA_cusolver_LIBRARY})
+  target_compile_definitions(sd_gpu PRIVATE SD_USE_THRUST SD_USE_GPU)
+  if(HIP)
```

Comment:
I think this is not the proper way to distinguish between usage of ROCm and CUDA libraries.

Reply:
we could test for `HIP_VERSION` or `HIP_COMPILER`, @mkuron do you have a suggestion?

Reply:
`HIP_VERSION` is the preferrable one as it is defined by FindHIP.cmake. `HIP` is defined by Espresso and should thus not be used inside the library.